hoot-core/src/main/cpp/hoot/core/algorithms/alpha-shape/AlphaShapeGenerator.cpp

```diff
@@ -106,76 +110,111 @@ std::shared_ptr<Geometry> AlphaShapeGenerator::generateGeometry(OsmMapPtr inputM
   LOG_VART(points.size());
 
   // create a complex geometry representing the alpha shape
-
-  std::shared_ptr<Geometry> cutterShape;
```

Comment:
Moved this variable to be a member variable so that it could be used in the thread.